GPU/Common/TextureReplacer.cpp

```diff
@@ -226,31 +226,50 @@ bool TextureReplacer::LoadIniValues(IniFile &ini, VFSBackend *dir, bool isOverri
 		ERROR_LOG(G3D, "Unsupported texture replacement version %d, trying anyway", version);
 	}
 
-	bool filenameWarning = false;
+	int badFileNameCount = 0;
 
 	std::map<ReplacementCacheKey, std::map<int, std::string>> filenameMap;
+	std::string badFilenames;
 
 	if (ini.HasSection("hashes")) {
-		auto hashes = ini.GetOrCreateSection("hashes")->ToMap();
+		auto hashes = ini.GetOrCreateSection("hashes")->ToVec();
```

Comment:
When will it make sense to map the same hash to two different filenames?  The level is still part of the key, so this would just overwrite those duplicates silently anyway?

Not sure I understand what the intent would be for:

1234567890abcdef_0 = image1.png
1234567890abcdef_0 = image2.png
1234567890abcdef_0 = image3.png

-[Unknown]

Reply:
The map was deduplicating the empty keys which were confusing my logging, which I added because of a theory that turned out to be wrong.